docs/en_US/Compression/v2_pruning_config_list.rst

```diff
@@ -0,0 +1,66 @@
+Pruning Config List Interpretation
+==================================
+
+The Keys in Config List
+-----------------------
+
+Each sub-config in the config list is a dict, and the scope of each setting (key) is only internal to each sub-config.
+If multiple sub-configs are configured with the same module, the later ones will overwrite the previous ones.
+
+op_types
+^^^^^^^^
+
+The type of the module specified in this sub-config.
+If ``op_names`` is not set in this sub-config, all modules in the model that satisfy the type will be selected.
+If ``op_names`` is set in this sub-config, the selected module should satisfy both type and name.
+
+op_names
+^^^^^^^^
+
+The name of the module specified in this sub-config.
+If ``op_types`` is set in this sub-config, the selected module should satisfy both type and name.
+
+op_partial_names
+^^^^^^^^^^^^^^^^
+
+This key is for the modules to be pruned with similar names. NNI will find all names in the model,
+find names that contain one of ``op_partial_names``, and append them into the ``op_names``.
+
+sparsity
+^^^^^^^^
+
+The sparsity ratio of each selected module.
+
+e.g., the ``sparsity`` is 0.8 means each selected module will mask 80% values on the weight.
+If ``layer_1`` (500 parameters) and ``layer_2`` (1000 parameters) are selected in this sub-config,
+then ``layer_1`` will be masked 400 parameters and ``layer_2`` will be masked 800 parameters.
+
+sparsity_per_layer
+^^^^^^^^^^^^^^^^^^
+
+Another name for ``sparsity``.
```

Comment:
which one do you recommend? "sparsity" or "sparsity_per_layer"?

Reply:
I think `sparsity_per_layer` is better in v2.

In v1, `sparsity` has both meaning `sparsity_per_layer` and `total_sparsity`, and in most cases, it means `sparsity_per_layer`. So for taking care of v1 users, we define `sparsity` same as `sparsity_per_layer`.

Reply:
so you recommend `sparsity_per_layer`, then we can make it clear. you can move the description from `sparsity` to `sparsity_per_layer`, then under `sparsity` you can say it has the same meaning as `sparsity_per_layer` but will be deprecated in future.

Reply:
That's a good suggestion, fixed it